src/uu/more/src/more.rs

```diff
@@ -187,11 +188,12 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
                 }
                 Ok(opened_file) => opened_file,
             };
+            let next_file_str = next_file.map(|f| f.to_string_lossy().into_owned());
             more(
                 InputType::File(BufReader::new(opened_file)),
                 length > 1,
-                file.to_str(),
-                next_file.copied(),
+                Some(&file.to_string_lossy()),
```

Comment:
I think you should fix `more` and `Pager`...

Reply:
i think it requires a bunch of changes, ok if I do that in a different PR ? 
(and i don't think i regressed the current state)